crates/cheatnet/tests/cheatcodes/replace_bytecode.rs

```diff
@@ -0,0 +1,111 @@
+use crate::assert_success;
+use crate::common::state::{build_runtime_state, create_cached_state};
+use crate::common::{call_contract, deploy_wrapper, felt_selector_from_name, get_contracts};
+use cairo_felt::Felt252;
+use cheatnet::runtime_extensions::forge_runtime_extension::cheatcodes::declare::declare;
+use cheatnet::state::CheatnetState;
+
+#[test]
+fn override_entrypoint() {
+    let mut cached_state = create_cached_state();
+    let mut cheatnet_state = CheatnetState::default();
+    let mut runtime_state = build_runtime_state(&mut cheatnet_state);
+
+    let contracts = get_contracts();
+
+    let class_hash_a = declare(&mut cached_state, "ReplaceBytecodeA", &contracts).unwrap();
+    let class_hash_b = declare(&mut cached_state, "ReplaceBytecodeB", &contracts).unwrap();
+    let contract_address =
+        deploy_wrapper(&mut cached_state, &mut runtime_state, &class_hash_a, &[]).unwrap();
+
+    let selector = felt_selector_from_name("get_const");
```

Comment:
Let's call this function before changing the class_hash to show that the returned value has changed